### PR TITLE
Laravel 11

### PR DESCRIPTION
**Pull request by matt-moellering**
Fixes #

## Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     },
     "require": {
         "php": "^8.0",
-        "illuminate/support": "^6.20 || ^7.0 || ^8.0 || ^9.0 || ^10.0",
+        "illuminate/support": "^6.20 || ^7.0 || ^8.0 || ^9.0 || ^10.0 || ^11.0",
```
Comment (atymic) [inline composer.json:38]:
```suggestion
        "illuminate/support": "^8.0 || ^9.0 || ^10.0 || ^11.0",
```

**Review by atymic — APPROVED** [review]
Can drop very old laravel versions out of support for a long time.

**Review by atymic — APPROVED** [review]

**Comment by PerryvanderMeer** [comment]
@atymic Any chance you can tag a new version with this included?

**Comment by atymic** [comment]
Tagged sorry

**Pull request merged**